src/main/java/core/basesyntax/Application.java

```diff
@@ -2,7 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // test your code here ...
-
+        Engine engine = new Engine("JZ-1", 600, 550, 9.5, 7);
+        Car toyota = new Car("Toyota", "Red", engine, 220, 1800);
+        Car clonedToyota = toyota.clone();
+        clonedToyota.setEngine(new Engine("JZ-2", 700, 550, 12.5, 6));
+        System.out.println(toyota.toString());
+        System.out.println(clonedToyota.toString());
```

Comment:
this is the task description:

1. create an engine; - done
2. create a car and assign an engine to it; - done
3. declare carClone variable by calling clone() on our existing car; - done
4. **change one of the parameters (fields) of engine in the original car; - not done**
5. print the parameters of both cars to the console. - done

let's fix the 4th point